CONTRIBUTING.md

```diff
@@ -42,6 +43,17 @@ Please submit bug reports as GitHub Issues. When submitting bug reports, please
 #### Getting Started
 Please refer to the [developer docs](./docs/dev/getting-started.md) for information on getting started with developing on `oc-mirror`.
 
+#### Debugging oc-mirror
+Since end of November 2023, and PR [#721](https://github.com/openshift/oc-mirror/pull/721), oc-mirror's v2 implementation is relying on [Buildah](https://github.com/containers/buildah) in order to
+build Cincinnati graph images. 
+
+Since oc-mirror needs to run rootless, it is now re-executing itself in a separate user namespace where buildah can have root privileges.
+
+This modifies the debugging process: In order to debug you should launch dlv debugger for oc-mirror in a user namespace: 
+```
+podman unshare dlv debug cmd/oc-mirror/main.go -- --v2 -c cfe-969.yml --from file:///home/skhoury/release docker://localhost:5000/cfe971/ --dest-tls-verify=false
```

Comment:
```suggestion
podman unshare dlv debug --headless --listen=:43987 cmd/oc-mirror/main.go -- --v2 -c cfe-969.yml --from file:///home/skhoury/release docker://localhost:5000/cfe971/ --dest-tls-verify=false
```

Once the command above is triggered it is possible to use delve to debug (either using dlv directly or attaching to it with a client).

If you use VSCode, it is possible to attach it to the dlv process running in the background, in order to do that it is necessary to add the following code to the configurations[] inside of the launch.json

```
{
    "name": "Attach Package",
    "type": "go",
    "debugAdapter": "dlv-dap",
    "request": "attach",
    "mode": "remote",
    "host": "localhost",
    "port": 43987,
},
{
    "name": "Attach Tests",
    "type": "go",
    "debugAdapter": "dlv-dap",
    "request": "attach",
    "mode": "remote",
    "host": "localhost",
    "port": 43987,
}

```
```suggestion
podman unshare dlv debug cmd/oc-mirror/main.go -- --v2 -c cfe-969.yml --from file:///home/skhoury/release docker://localhost:5000/cfe971/ --dest-tls-verify=false
```